mod_modular_vanilla/hooks/ai/tactical/behavior.nut

```diff
@@ -0,0 +1,13 @@
+::ModularVanilla.MH.hook("scripts/ai/tactical/behavior", function(q) {
+	// MV: Changed
+	// Add skill_container events to allow skills to modify the queried value
+	q.queryTargetValue = @(__original) function( _entity, _target, _skill = null )
+	{
+		local ret = __original(_entity, _target, _skill);
+
+		ret = _entity.getSkills().onQueryTargetValue(_entity, _target, _skill, ret);
+		ret = _target.getSkills().onQueryTargetValue(_entity, _target, _skill, ret);
```

Comment:
I like combining both suggested functions into one.
If I understand that correctly then that decision would just cause every skill implementation of this event to start with:
```
if (_entity.getID() == this.getContainer().getActor().getID())
{
}
else
{
}
```
Because this event is currently only fired twice per queryTargetValue calculation and there are always only these two cases. Either _entity is me, or _target is me
Do I understand that correctly?

Reply:
Yes that is correct. In MSU I wanted to do it like this for `onGetHitFactors` but we had already released `onGetHitFactors` before I realized that we should add it for target as well, and there was no backwards compatible way to introduce that without making a separate function for being target. But in this case, we can do this as it is a new feature.